src/Layouts/JumpstartUiLayouts.php

```diff
@@ -51,6 +78,7 @@ public function submitConfigurationForm(array &$form, FormStateInterface $form_s
     array_walk($classes, 'trim');
     $this->configuration['extra_classes'] = implode(' ', array_filter($classes));
     $this->configuration['centered'] = $form_state->getValue('centered') ? 'centered-container' : NULL;
+    $this->configuration['columns'] = $form_state->getValue('columns') ?? 'default';
```

Comment:
during a submit we can be 100% sure `columns` will be a set value.
```suggestion
    $this->configuration['columns'] = $form_state->getValue('columns');
```